src/delphi/fmx.castlecontrol.pas

```diff
@@ -14,7 +14,7 @@
 }
 
 { Control with OpenGL context on a Delphi FMX form. }
-unit Fmx.CastleControl;
+unit fmx.castlecontrol;
```

Comment:
Please apply similar comments as for VCL counterpart, so keep unit name `PascalCase`, so `Fmx.CastleControl`. And just remove (not only comment-only) the design-time pieces moved.